input/ig.xml

```diff
@@ -300,34 +308,34 @@
 		<!--LOCATION EXAMPLES-->
 		<resource>
 			<reference>
-				<reference value="Location/au-hospital-pharm-out"/>
+				<reference value="Location/bobrester-medical-center"/>
 			</reference>
-			<name value="Location - Friends Pharmacy"/>
-			<description value="Friends Pharmacy (outpatient) - part of AU Hospital Site"/>
+			<name value="Location - Tindal Building"/>
+			<description value="Tindal Building, Bobrester Medical Centre"/>
 			<exampleCanonical value="http://hl7.org.au/fhir/core/StructureDefinition/au-core-location"/>
 		</resource>
 		<resource>
 			<reference>
-				<reference value="Location/au-hospital-sw-physio"/>
+				<reference value="Location/patient-home"/>
 			</reference>
-			<name value="Location - Physiotherapy Offices - AU Hospital"/>
-			<description value="Second floor, South Wing, Physiotherapy Offices - part of AU Hospital Site"/>
+			<name value="Location - Patient home"/>
+			<description value="Patient home as kind of location (not a specific patient's location)."/>
 			<exampleCanonical value="http://hl7.org.au/fhir/core/StructureDefinition/au-core-location"/>
 		</resource>
 		<resource>
 			<reference>
-				<reference value="Location/bobrester-medical-center"/>
+				<reference value="Location/murrabit-hospital"/>
 			</reference>
-			<name value="Location - Tindal Building"/>
-			<description value="Tindal Building, Bobrester Medical Centre"/>
+			<name value="Location - Murrabit Public Hospital"/>
+			<description value="Murrabit Public Hospital"/>
 			<exampleCanonical value="http://hl7.org.au/fhir/core/StructureDefinition/au-core-location"/>
 		</resource>
 		<resource>
-			<reference>
-				<reference value="Location/patient-home"/>
+		<reference>
+				<reference value="Location/mitchells-hill-audiology"/>
 			</reference>
-			<name value="Location - Patient home"/>
-			<description value="Patient home as kind of location (not a specific patient's location)."/>
+			<name value="Location - Murrabit Public Hospital"/>
```

Comment:
Wrong name, this entry is for Mitchells Hill Audiology

Reply:
Done